source/blog/2016-04-08-new-in-rdo-repos-one-size-doesn-t-fit-all.html.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Newbie in RDO: one size doesn't fit all"
+author: amoralej
+tags: openstack, rdo, repos
+date: 2016-04-08 19:50:56 CEST
+---
+
+As a new contributor to the RDO project, one of my first tasks was to understand what is actually being produced by the project. According to [RDO faq](https://www.rdoproject.org/rdo/faq/), the main goal is to maintain a freely-available, community-supported distribution of OpenStack based on RPM packages able to run Red Hat Enterprise Linux, CentOS, Fedora, and their derivatives. But the RDO community is diverse and different use patterns have been found over the time with different requirements:
+
+- Organizations running production OpenStack clouds with community support require a stable distribution.
+- Organizations running production environments with community or enterprise-grade distributions require sometimes an up-to-date distribution to test recent functionalities and plan the future of their cloud deployments.
+- Developers require up-to-date distribution to test their changes and run their CI jobs.
+- Packagers need to have full control over the packaging process.
+
+Providing a suitable distribution for all these scenarios is not an easy task and the RDO community ended up with two different distribution set of repos targetting different purposes:
+
+### RDO Trunk repos
+Provide RPM package repositories for OpenStack projects as close as possible to the vanilla upstream code in order to test any change and catch any bug or impact on packaging as soon as possible. These packages are built when a commit is merged in upstream master or any of the supported stable branches. Once a build is done, a set of CI jobs is passed to validate installation and basic functionalities. As part of RDO Trunk we provide different repos for CentOS 7 under http://buildlogs.centos.org/centos/7/cloud/x86_64/ :
+
+- **rdo-trunk-&lt;branch>:** provides the latest snapshot of packages successfully built (i.e. rdo-trunk-mitaka).
+- **rdo-trunk-&lt;branch>-tested:** provides the latest snapshot of packages built and with CI jobs successfully passed (i.e. rdo-trunk-master-tested).
+- **rdo-trunk-master-tripleo:** provides the latest snapshot of packages tested by specific TripleO CI jobs on master branch.
+
+Currently (apr '16'), &lt;branch> can be master, mitaka, liberty or kilo.
```

Comment:
@amoralej update this line before publishing to include only master, mitaka and liberty.
Kilo is EOL now.